.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Package Properties
+
+env:
+  PRJ_NAME: "citus"
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      fancy_version_no:
+        description: 'If fancy versioning is needed, set value greater than 1 e.g 2'
+        default: '1'
+      microsoft_email:
+        description: 'Email to be written on changelogs'
+        default: 'gindibay@microsoft.com'
+      name:
+        description: 'Name to be written on changelogs'
+        default: 'Gurkan Indibay'
+jobs:
+  update_package_properties:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone tools branch
+        run: git clone --branch update_properties_fixes https://github.com/citusdata/tools.git
```

Comment:
Do we want to keep using this custom branch?

Reply:
Fixed